examples/named_entity_recognition/ner_wikigold_transformer.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Can we rename the maxLen arg to max_len to follow the same variable naming convention?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Changed. I just cannot control to use hump naming style. </p>